test/integrationtests/http_client_helper.go

```diff
@@ -0,0 +1,32 @@
+package integrationtests
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+var defaultClient = &http.Client{}
```

Comment:
do we need this or can we use the http.DefaultClient ?

Reply:
You are right ! We dont need it, removed it [here](https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/97/commits/4b66c329c1a307d02c490ed9a6539e96b94e49e5)